product_manufacturer_add_supplierinfo/README.rst

```diff
@@ -0,0 +1,46 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+==============
+Product manufacturer add supplier info
+==============
+
+Extended product template. This module allows to automatically create a product
+supplier from the product manufacturer information.
+
+Bug Tracker
```

Comment:
This is not an OCA module. You don't have to follow that template. Also, you missed the important part 'usage'. Try something like: to use this module you have to go to... etc.